ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/console/OutputConsoleViewImpl.java

```diff
@@ -317,6 +317,11 @@ public void print(String text, boolean carriageReturn) {
 
   @Override
   public void print(final String text, boolean carriageReturn, String color) {
+
+    if (consoleLines.getElement().getChildCount() > 500) {
```

Comment:
It is better to have a constant but not a magic number ^)

Reply:
I would say this change would not make this fast and dirty fix less dirty.